File: computer_networks/bitStuffing.c
#include<stdio.h>
#include<stdlib.h>
#include<string.h>
void main(){
    char a[50];
    int t,x=0;
    printf("Enter the data ");
    scanf("%s",a);
    t=strlen(a);
    printf("asner\n%d ",t);
    for(int i=0;i<t;i++){
        printf("%c",a[i]);
        if(a[i]=='1')
        x++;
        if(a[i]=='0')
        x=0;
        if(x==5){
            printf("0");
            x=0;
        }
       
    }
}

File: Computer_networks/bitStuffing.py
#bitstuffing
a=input("enter the data: ")
print("the stuffed data is: ")
oc=0
for i in a:
    print(i,end=" ")
    if (i=="1"):
        oc+=1
    else:
        oc=0
    if oc==5:
        print(0,end=" ")
        
        
